tools/cldr-code/src/main/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -2,43 +2,73 @@
 
 import java.util.HashSet;
 import java.util.Set;
+import java.util.concurrent.ExecutionException;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import org.unicode.cldr.util.GrammarInfo;
+import org.unicode.cldr.util.Level;
 import org.unicode.cldr.util.Organization;
 import org.unicode.cldr.util.RegexUtilities;
 import org.unicode.cldr.util.StandardCodes;
+import org.unicode.cldr.util.SupplementalDataInfo;
 
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.ImmutableSortedSet;
 
+/**
+ * This class manages the Limited Submission process.
+ * 
+ * TODO: see https://unicode-org.atlassian.net/browse/CLDR-15230 for TODOs here
+ */
 public final class SubmissionLocales {
+    // TODO:  On the use of Locales.txt here, see https://unicode-org.atlassian.net/browse/CLDR-14838
+    /**
+     * This is the 'raw' list from Locales.txt
+     */
+    public static final Set<String> CLDR_LOCALES = StandardCodes.make()
+        .getLocaleToLevel(Organization.cldr).keySet();
 
-    public static final Set<String> CLDR_LOCALES = StandardCodes.make().getLocaleToLevel(Organization.cldr).keySet();
+    /**
+     * This is the 'special' list from Locales.txt
+     */
+    public static final Set<String> SPECIAL_ORG_LOCALES = StandardCodes.make()
+        .getLocaleToLevel(Organization.special).keySet();
 
     /**
      * Non-CLDR Locales, but consistently have high level of engagement from volunteers to keep at modern level.
      * Reevaluate for each release based on meeting 95+% of modern, moderate, and basic coverage
      */
     public static Set<String> HIGH_LEVEL_LOCALES = ImmutableSet.of(
-        "chr",  // Cherokee
-        "gd",   // Scottish Gaelic, Gaelic
-        "fo",   // Faroese
-        "kok",  // Konkani
-        "pcm",  // Nigerian Pidgin
-        "ha",   // Hausa
-        "hsb",  // Upper Sorbian
-        "dsb",  // Lower Sorbian
-        "yue_Hans",   // Cantonese (Simplified)
-        "to"    //  Tongan
-        );
+        // Note: ALL of these were found in Locales.txt under cldr.
+        "chr", // Cherokee
+        "gd", // Scottish Gaelic, Gaelic
+        "fo", // Faroese
+        "kok", // Konkani
+        "pcm", // Nigerian Pidgin
+        "ha", // Hausa
+        "hsb", // Upper Sorbian
+        "dsb", // Lower Sorbian
+        "yue_Hans", // Cantonese (Simplified)
+        "to" //  Tongan
+    );
 
     public static final Set<String> CLDR_OR_HIGH_LEVEL_LOCALES = ImmutableSet.<String>builder()
```

Comment:
I think the HLL should replaced by Org=special (which is itself a subset of Org=cldr). That way it will be kept up to date. Is there a reason not to?

Reply:
HLL as it is now is essentially dead code, adding no value. I already remove special for missing. So if anything I'd just delete all references to HLL.

Reply:
for future cleanup such as CLDR-15230